src/components/modules/CodeRecovery/CodeRecovery.interactor.vue

```diff
@@ -11,12 +10,12 @@
 </template>
 
 <script>
-import OtpRecovery from '@/components/modules/OtpRecovery';
+import RecoveryView from './modules/RecoveryView';
```

Comment:
Please, don't use view and other prefixes in naming after import. If module is changed layer, we will need rename template, component section and etc. After from it's ok